spec/spec.md

```diff
@@ -1462,6 +1498,23 @@ The following JSON Schema Draft 7 definition summarizes the rules above:
 
 </tab-panels>
 
+## Credential Format Designations
+
+Within the _Presentation Exchange_ specification, there are numerous sections 
+where Verifiers and Holders convey what credential variants they support and 
+are submitting. The following are the normalized references used within the 
+specification:
+
+- `vc`, `vp` - these supported formats are defined in the 
```

Comment:
We should consider also allowing `ldp`, representing any LD-Proof, just as we allow `jwt`.